Cookbook.Infrastructur/Persistance/RecipebookRepository.cs

```diff
@@ -0,0 +1,283 @@
+﻿using Cookbook.Application.Database;
+using Cookbook.Domain.Models;
+using Dapper;
+using System.Data;
+
+namespace Cookbook.Infrastructur.Persistance //restructure DB to use rowID as FK instead of slug
```

Comment:
//restructure DB to use rowID as FK instead of slug => yes please

Reply:
lol